backup_and_restore/application_backup_and_restore/installing/about-oadp-data-mover.adoc

```diff
@@ -6,17 +6,33 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-{oadp-short} includes a built-in Data Mover that you can use to move Container Storage Interface (CSI) volume snapshots to a remote object store. The built-in Data Mover allows you to restore stateful applications from the remote object store if a failure, accidental deletion, or corruption of the cluster occurs. It uses xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-about-kopia.adoc#oadp-about-kopia[Kopia] as the uploader mechanism to read the snapshot data and write to the unified repository.
+{oadp-first} includes a built-in Data Mover that you can use to move Container Storage Interface (CSI) volume snapshots to a remote object store. The built-in Data Mover allows you to restore stateful applications from the remote object store if a failure, accidental deletion, or corruption of the cluster occurs. It uses xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-about-kopia.adoc#oadp-about-kopia[Kopia] as the uploader mechanism to read the snapshot data and write to the unified repository.
 
 {oadp-short} supports CSI snapshots on the following:
 
 * {odf-full}
 * Any other cloud storage provider with the Container Storage Interface (CSI) driver that supports the Kubernetes Volume Snapshot API
 
-[IMPORTANT]
-====
+// ** suggest removing the admonition and replacing with a clear section **
+//
+// [IMPORTANT]
+// ====
+// The {oadp-short} built-in Data Mover, which was introduced in {oadp-short} 1.3 as a Technology Preview, is now fully supported for both containerized and virtual machine workloads.
+// ====
+
+[id="oadp-data-mover-support_{context}"]
+== Data Mover support
+
 The {oadp-short} built-in Data Mover, which was introduced in {oadp-short} 1.3 as a Technology Preview, is now fully supported for both containerized and virtual machine workloads.
-====
+
+Backups taken with {oadp-short} 1.1 or {oadp-short} 1.2 using the Data Mover feature cannot be restored using {oadp-short} 1.3 and later. Therefore, it is not supported.
+
+The Data Mover backups taken with {oadp-short} 1.3 can be restored using {oadp-short} 1.3, 1.4, and later. This is supported.
+
+In {oadp-short} 1.1 and 1.2, Data Mover does not support backups and restores. Data Mover in  {oadp-short} 1.1 and 1.2 remains a Technology Preview so it is therefore not supported. 
```

Comment:
I think we can remove this line (32).  Or  replace it with 
OADP-1.1 and OADP-1.2 are no longer supported.  The DataMover feature in  OADP-1.1 or OADP-1.2 was tech preview and never supported.  DataMover backups taken with OADP-1.1 or OADP-1.2 can not be restored on later versions of OADP.  ( Your call )